web/src/pages/index.tsx

```diff
@@ -14,57 +14,72 @@ import CallToActionImage, {
 import { Layout } from '../components/layouts';
 
 const filterType = (obj, type) => {
-  return obj.allMdx.edges
+  return obj.edges
     .filter((edge) => edge.node?.frontmatter)
     .filter((edge) => edge.node.frontmatter.type === type)
     .map((edge) => edge.node.frontmatter.homePage);
 };
 
-const Homepage: React.FC = () => {
-  const data = useStaticQuery(graphql`
-    query AllQuery {
-      allMdx(sort: { fields: [frontmatter___id], order: ASC }) {
-        edges {
-          node {
-            frontmatter {
-              type
-              homePage {
-                title
-                linkTo
-                linkText
-                image
-                hoverImage
-                description
-                imageStyle {
-                  ml
-                }
+export const query = graphql`
```

Comment:
No longer using `useStaticQuery` 🤔

Does this still turn into static data or?

Reply:
@cobraz It is a localization library 'hook', it parses data from export const